static/app/chartcuterie/performance.tsx

```diff
@@ -23,11 +31,55 @@ function modifyOptionsForSlack(options: Omit<LineChartProps, 'series'>) {
     visualMap: options.options?.visualMap,
   };
 }
+type FunctionRegressionChartData = Omit<
+  EventBreakpointChartData,
+  'chartType' | 'percentileData'
+> & {rawResponse: any};
 
 performanceCharts.push({
   key: ChartType.SLACK_PERFORMANCE_ENDPOINT_REGRESSION,
-  getOption: (data: EventBreakpointChartData) => {
-    const {chartOptions, series} = getBreakpointChartOptionsFromData(data, theme);
+  getOption: (data: EndpointRegressionChartData) => {
+    const param = {
+      ...data,
+      chartType: ChartType.SLACK_PERFORMANCE_ENDPOINT_REGRESSION,
+    };
+
+    const {chartOptions, series} = getBreakpointChartOptionsFromData(param, theme);
```

Comment:
i didn't want to pass it to `getBreakpointChartOptionsFromData` because I would [prop drill](https://kentcdodds.com/blog/prop-drilling) bunch of times for the Sentry Charts.